include/cantera/kinetics/GasKinetics.h

```diff
@@ -89,6 +89,15 @@ class GasKinetics : public BulkKinetics
     Rate1<Plog> m_plog_rates;
     Rate1<ChebyshevRate> m_cheb_rates;
 
+    //! Vector of generic reaction rates
+    std::vector<shared_ptr<RxnRate>> m_rxn_rates;
+
+    //! Vector of specialized arrhenius reaction rates
+    std::vector<ArrheniusRate> m_arrhenius_rates;
+
+    std::map<size_t, size_t> m_rxn_indices;
+    std::map<size_t, size_t> m_arrhenius_indices;
+
```

Comment:
These would be replaced by `MultiRate` objects; but no objection to moving those to `Kinetics`.

Reply:
Great. I think doing that would (eventually) eliminate the need for most specializations of `Kinetics` other than `BulkKinetics` and `InterfaceKinetics`.

Reply:
I believe it may make sense to keep two separate templated `MultiRateBase` classes that handle `BulkKinetics` and `InterfaceKinetics`, respectively. As such, I added a `m_bulk_rates` to `BulkKinetics`, and left interface-specific routines for a later time.